### DISCUSSION TITLE
v3 layout on going?

### DISCUSSION THREAD

**Discussion by MacPaille** [firefly-iii/firefly-iii#5469] [General]
Hi @JC5 
With this new warning for the v2 layout being deprecated when can we expect to see the first version of layout v3? Also Maybe adding a common ticket for people to provide input/comments related to v3 would be nice.

Don't know how it will look but I'm impatient to see it 😄 

BTW I totally forgot with my last ticket, best wishes for 2022 and FFIII !

**Comment by JC5** [comment]
Hi!

I don't know yet. The whole reason the v2 layout is stalling because it takes a lot of time and effort to build the smallest things. That's my own fault, really. I started kind of unprepared.

The v3 layout will not be released until I'm reasonably ahead with pages, features and code. I want to prevent another situation where I over promise and under deliver. Before I release anything, I'd like to build a dashboard, some create/edit pages (transactions + accounts for example), lists for all models (page handling, caching, etc), and flexible caching for data coming from the API. After that, maybe.

You all the best for 2022 as well!
